experimental/AlgebraicStatistics/src/GaussianGraphicalModels.jl

```diff
@@ -374,26 +307,25 @@ and then eliminating all variables `k[i,j]` where $ K =$ `concentration_matrix(M
 ## Examples
 
 ```jldoctest
-julia> M = graphical_model(graph_from_edges([[1,2], [2,3]]), gaussian_ring(3))
-Gaussian graphical model on an undirected graph with edges:
-(1, 2), (2, 3)
+julia> M = gaussian_graphical_model(graph_from_edges([[1,2], [2,3]]))
+Gaussian Graphical Model on a Undirected graph with 3 nodes and 2 edges
 
 julia> vanishing_ideal(M)
 Ideal generated by
   -s[1, 2]*s[2, 3] + s[1, 3]*s[2, 2]
 ```
 """
-function vanishing_ideal(M::GraphicalModel{Graph{Undirected}, GaussianRing})
+function vanishing_ideal(M::GaussianGraphicalModel{Undirected, Nothing})
```

Comment:
@bkholler or @taboege does it make sense for this to be the vanishing ideal function in this case?
or is there a way to setup this vanishing ideal function to also use the parametrization?

i can Imagine that if we set this up to go through the parametrization nothing needs to change for the case when the graph is colored?